src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -316,12 +322,18 @@ public static class TextCatalog
                 outputColumnName, inputColumnName, ngramLength, skipLength, useAllLengths, maximumNgramsCount, weighting);
 
         /// <summary>
-        /// Produces a bag of counts of ngrams (sequences of consecutive words) in <paramref name="inputColumnNames"/>
-        /// and outputs bag of word vector as <paramref name="outputColumnName"/>
+        /// Create a <see cref="WordHashBagEstimator"/>, which maps the multople columns specified in <paramref name="inputColumnNames"/>
+        /// to a vector of counts of n-grams in a new column named <paramref name="outputColumnName"/>.
         /// </summary>
-        /// <param name="catalog">The text-related transform's catalog.</param>
-        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnNames"/>.</param>
-        /// <param name="inputColumnNames">Name of the columns to transform.</param>
+        /// <remarks>
+        /// <see cref="WordBagEstimator"/> is different from <see cref="NgramExtractingEstimator"/> in that the former
+        /// tokenizes text internally and the latter takes tokenized text as input.
+        /// </remarks>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnNames"/>.
+        /// This column's data type will be known-size vector of <see cref="System.Single"/>.</param>
+        /// <param name="inputColumnNames">Names of the multiple columns to take the data from.
+        /// This estimator operates over vector of text.</param>
```

Comment:
>text [](start = 51, length = 4)

cref=system.String?

Reply:
If you take this change, please update all functions

---
In reply to: [277113589](https://github.com/dotnet/machinelearning/pull/3440#discussion_r277113589) [](ancestors = 277113589)